mariamihaelafainarea/assignment2/LowestAncestor

```diff
@@ -0,0 +1,100 @@
+    /*
+     * To change this license header, choose License Headers in Project Properties.
+     * To change this template file, choose Tools | Templates
+     * and open the template in the editor.
+     */
+
+    /**
+     *
+     * @author maria
+     */
+    public class LowestAncestor<T> {
+
+        Node first;
+        //these parameters help me find
+        //if the two values searched were found
+
+        public class Node {
```

Comment:
You are defining the same inner class Node in both LowerAncestor and PrintPath classes: why not just define it once outside and use it in both classes?